Patch-Source: https://gitlab.alpinelinux.org/alpine/abuild/-/merge_requests/60
--
From 55b66b3ad702ba673384126d98026043cdf949ef Mon Sep 17 00:00:00 2001
From: Leo <thinkabit.ukim@gmail.com>
Date: Tue, 7 Jul 2020 13:37:31 -0300
Subject: [PATCH] newapkbuild: use meson-abuild helper instead of meson

abuild-meson is a shellscript that wraps around meson and passes options
meant for packaging software that uses meson, for Alpine Linux.
---
 newapkbuild.in | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/newapkbuild.in b/newapkbuild.in
index c58d519..d7a2eb5 100644
--- a/newapkbuild.in
+++ b/newapkbuild.in
@@ -78,12 +78,7 @@ build_meson() {
 # TODO For cross compilation a cross_file needs to be created.
 
 	cat >>APKBUILD<<__EOF__
-	meson \\
-		--prefix=/usr \\
-		--sysconfdir=/etc \\
-		--mandir=/usr/share/man \\
-		--localstatedir=/var \\
-		--buildtype=plain \\
+	abuild-meson \\
 		. output
 	meson compile \${JOBS:+-j \${JOBS}} -C output
 __EOF__
-- 
GitLab

